os/kernel/init/os_bringup.c

```diff
@@ -279,16 +279,7 @@ static inline void os_do_appstart(void)
 #endif
 
 #ifdef CONFIG_TASK_MANAGER
-#define TASKMGR_STACK_SIZE 2048
-#define TASKMGR_PRIORITY 200
-
-	svdbg("Starting task manager\n");
 	task_manager_drv_register();
```

Comment:
preapp launches task manager. But this driver registeration is executed after preapp. Let's move this before preapp launching.